src/main/java/com/archyx/aureliumskills/api/AureliumAPI.java

```diff
@@ -193,6 +194,18 @@ public static int getSkillLevel(UUID playerId, Skill skill) {
         }
     }
 
+    /**
+     * Gets the total skill level of a player
+     * @return the total skill level of a player, will display number of skills available if player does not have a skills profile
+     */
+    public static int getTotalLevel(Player player) {
+        int totalLevel = 0;
+        for(Skill skill : Skills.values()){
```

Comment:
This should use `plugin.getSkillRegistry().getSkills()` to account for future possible custom skills.

Reply:
Also add space after for and before open curly brace.